src/Brick/Widgets/Core.hs

```diff
@@ -843,9 +843,8 @@ withAttr an p =
       withReaderT (ctxAttrNameL .~ an) (render p)
 
 -- | Update the attribute map while rendering the specified widget: set
--- its new default attribute to the one that we get by looking up the
```

Comment:
The original comment referred to some attribute name, but this function is not taking any attribute name as input..